feature-group-definitions/check-visibility.yml

```
@@ -1,4 +1,5 @@
 name: checkVisibility()
+description: "The `checkVisibility()` method checks if an element is visible. For example, it checks whether an element has the style `display: none`."
```

The name even was contentious because this is hard to summarize, but maybe should quote the spec "provides a set of simple checks for whether an element is potentially 'visible', with options to define which precise notion of 'visibility' is desired."

---

Noting that there are multiple kinds of visibility sounds good. Trying to match the guidelines in https://github.com/web-platform-dx/web-features/pull/789:

```suggestion
description: "The `checkVisibility()` method checks if an element is potentially visible, with optional parameters for the kinds of visibility to check. For example, it checks whether the element has the style `display: none`, but can also check for `visibility: hidden`."
```

I reduced it to just one example from the default checks and the optional checks to resist the temptation of making it complete.